src/sidecar/sidecarManager.ts

```diff
@@ -447,7 +447,12 @@ export class SidecarManager {
       fs.writeFileSync(SIDECAR_LOGFILE_PATH, "");
     }
 
-    this.logTailer = new Tail(SIDECAR_LOGFILE_PATH);
+    try {
+      this.logTailer = new Tail(SIDECAR_LOGFILE_PATH);
+    } catch (e) {
+      sidecarOutputChannel.appendLine(`Failed to tail sidecar log file: ${e}`);
```

Comment:
Should this also include `SIDECAR_LOGFILE_PATH` if users want to manually tail it in a terminal (since we won't get down to the next `.appendLine()` where it's normally shown)?